check/check.js

```diff
@@ -77,6 +78,15 @@ module.exports = (fields, locations, message) => {
     return middleware;
   };
 
+  middleware.customSanitizer = (sanitizer, ...options) => {
```

Comment:
To keep the same interface as the `.custom()` validator method, this should not receive any options.